apps/app/src/pages/Account/Settings.tsx

```diff
@@ -34,6 +36,7 @@ const AccountQuery = graphql(`
       ...AccountChangeName_Account
       ...AccountChangeSlug_Account
       ...PlanCard_Account
+      # ...AccountVercel_Account
```

Comment:
Are you sure that "#" will add comment ?

Reply:
Yeah, see the top of file